2-farm/lessons/1-predict-plant-growth/README.md

```diff
@@ -264,7 +264,11 @@ For these, find if there are sensors that can measure them. What about actuators
 Read more on digital agriculture on the [Digital Agriculture Wikipedia page](https://wikipedia.org/wiki/Digital_agriculture). Also read more about precision agriculture the [Precision Agriculture Wikipedia page](https://wikipedia.org/wiki/Precision_agriculture).
 
 The full growing degree days calculation is more complicated than the simplified one given here. Read more about the more complicated equation and how to deal with temperatures below the baseline on the [Growing Degree Day Wikipedia page](https://wikipedia.org/wiki/Growing_degree-day).
-
+<br>
+<a href="https://www.youtube.com/watch?v=KIEOuKD9KX8" target="_blank">Farming Technologies</a>
+<br>
+<a href="https://www.youtube.com/watch?v=AmVNwilQJvc" target="_blank">China's Farming</a>
+<br>
```

Comment:
These are long videos, so would be better placed in the review and self study section with a more detailed description, following the style of other items in that section